generatehtml.py

```diff
@@ -6,15 +6,32 @@
 import codecoverage
 
 
-def generate_html(data_folder):
+def generate_html(data_folder, ignore_third_party=True):
     with open('{}/diff.json'.format(data_folder), 'r') as report:
         parsed_json = json.load(report)
 
     file_obj = open('{}/output.info'.format(data_folder), 'w')
 
     source_files = parsed_json['source_files']
     file_obj.write('TN\n')
+
+    third_party_paths = []
```

Comment:
No need to define it to an empty array here, it will always be overwritten at line 20.